tests/injection/outjection.py

```diff
@@ -0,0 +1,33 @@
+# Copyright Metatype OÜ, licensed under the Mozilla Public License Version 2.0.
+# SPDX-License-Identifier: MPL-2.0
+
+from typegraph import typegraph, Policy, t, Graph
+from typegraph.runtimes import RandomRuntime, DenoRuntime
+
+
+@typegraph()
+def outjection(g: Graph):
+    deno = DenoRuntime()
+    random = RandomRuntime(seed=1)
+
+    g.expose(
+        Policy.public(),
+        randomUser=deno.identity(t.struct()).extend(
+            {
+                "id": t.uuid().from_random(),
+                "age": t.integer().set(19),
+                "email": t.email().from_context("user_email"),
+                "password": t.string().from_secret(
```

Comment:
tangent: do we have support or will we want to add support for optional injetions? I.e. only inject if no value is found on output?

Reply:
Interesting... We can explore that...

Reply:
https://linear.app/metatypedev/issue/MET-780/optional-injections